### PR TITLE
Create a record object

### PR DESCRIPTION
**Pull request by gingerben93**
Ok nick i made charges a property just for you!

### CODE REVIEW

File: src/backend/expungeservice/models/record_object.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from expungeservice.models.case import Case
+
+from decimal import Decimal
```
Comment (NickSchimek) [inline src/backend/expungeservice/models/record_object.py]: Can you remove both of these imports? They're currently not being used.

File: src/backend/tests/factories/record_factory.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from expungeservice.models.record_object import Record
+from expungeservice.models.charge import Charge
+from tests.factories.case_factory import CaseFactory
```
Comment (NickSchimek) [inline src/backend/tests/factories/record_factory.py]: Can you remove the charge and casefactory import?

File: src/backend/tests/models/test_record.py [hunk 3]
```diff
@@ -1,34 +1,51 @@
 import unittest
-
-from expungeservice.models.case import Case
-from expungeservice.models.record import Record
+from expungeservice.models.record_object import Record
```
Comment (NickSchimek) [inline src/backend/tests/models/test_record.py]: Can you remove this Record import? I don't see it being used.

File: src/backend/tests/models/test_record.py [hunk 4]
```diff
@@ -1,34 +1,51 @@
...
+    def test_num_cases(self):
+        
+        assert len(self.record.charges) == 3
+        
+    def test_charges_index_0(self):
+
+        assert self.record.charges[0] == self.charge_zero
+
+    def test_charges_index_1(self):
+        record = RecordFactory.create([self.case_zero, self.case_one])
```
Comment (NickSchimek) [inline src/backend/tests/models/test_record.py]: This temp var is not being used. Can this line be removed also?

File: src/backend/tests/models/test_record.py [hunk 5]
```diff
@@ -1,34 +1,51 @@
...
+
+        assert self.record.charges[1] == self.charge_one
+
+    def test_charges_index_2(self):
+        record = RecordFactory.create([self.case_zero, self.case_one])
```
Comment (NickSchimek) [inline src/backend/tests/models/test_record.py]: This var is also not being used. Can it be removed too?

**Comment by NickSchimek** [comment]
Thanks @gingerben93 the tests look great!
This can be merged although I'd suggest removing any unused code.

**Comment by gingerben93** [comment]
Ok should be ready to go!

**Pull request merged**